generator/generator_tests/geo_objects_tests.cpp

```diff
@@ -0,0 +1,125 @@
+#include <indexer/classificator.hpp>
```

Comment:
please change to "" and move to other indexer/ includes

Reply:
++ removed it completely